Amazon Scraper/scrape.py

```diff
@@ -0,0 +1,33 @@
+import csv
+import pandas as pd
+from bs4 import BeautifulSoup
+import requests
+import time
+def scraper():
```

Comment:
Give two blank spaces between the last import statement and the function definition